cdxev/set.py

```diff
@@ -192,20 +192,22 @@ def _should_overwrite(
                 "Use the --force option to overwrite."
             ),
         )
+    else:  # pragma: no cover
+        if _prompt_for_overwrite(property, component_id):
+            logger.debug(
+                f'Overwriting "{property}" on component "{component_id}" due to user choice.'
+            )
```

Comment:
Is this information _really_ required? I mean, you will already get the debug-information that the property will be set, feels kind of redundant as this happens through the prompt, i.e. I know that I choose "y"

Reply:
Can never have enough debug logging 🤷 😛 

Honestly, I'm fine removing it but its not like that hasn't always been there. I only refactored the code very lightly to remove it from coverage stats.